snapshottest/module.py

```diff
@@ -254,6 +259,23 @@ def assert_match(self, value, name=''):
     def save_changes(self):
         self.module.save()
 
+    def clear_ignore_keys(self, data):
+        if isinstance(data, dict):
+            for key, value in data.items():
+                if key in self.ignore_keys:
+                    data[key] = None
+                elif isinstance(value, dict):
```

Comment:
```suggestion
                elif isinstance(value, (dict, list)):
```
This change should make it so that dicts that contain lists of dicts can be correctly ignored.